modules/combined/tests/creep_plasticity_combined/combined_stress_prescribed.i

```diff
@@ -0,0 +1,159 @@
+#
+# 1x1x1 unit cube with time-varying pressure on top face
+#
+# The problem is a one-dimensional creep analysis.  The top face has a
+#    pressure load that is a function of time.  The creep strain can be
+#    calculated analytically.  There is no lsh plasticity.
```

Comment:
Does this comment mean that there is no plasticity active under these loading conditions?  There is a plasticity model.